nomad/leader.go

```diff
@@ -19,6 +20,15 @@ const (
 	// unblocked to re-enter the scheduler. A failed evaluation occurs under
 	// high contention when the schedulers plan does not make progress.
 	failedEvalUnblockInterval = 1 * time.Minute
+
+	// failedEvalFollowUpBaseLineWait is the minimum time waited before retrying
+	// a failed evaluation.
+	failedEvalFollowUpBaseLineWait = 1 * time.Minute
+
+	// failedEvalFollowUpWaitRange defines the the range of additional time from
+	// the minimum in which to wait before retrying a failed evaluation. A value
+	// from this range should be selected using a uniform distribution.
+	failedEvalFollowUpWaitRange = 9 * time.Minute
```

Comment:
We might consider making these threaded through via config as well. I can see wanting to tune this potentially.